generate/generate-metadata.sh

```diff
@@ -49,16 +49,12 @@ for src in dev compliance-tool; do
   fi
   
   # sign
-  if test -z `which xmlsectool`; then
```

Comment:
Why was this removed?

Reply:
because this runs with in a docker container where we are guaranteed to have  xmlsectool... Also the the following code targeted homebrew which doesn't run so well on linux or in docker.

Reply:
Might be nice to leave it in though in case someone wants to run it without docker though?

Reply:
Then I think it should just dump out that it needs xmlsectool and maybe information on how to get it rather than trying to run brew.

Reply:
Yep, fair do's.

Reply:
I have put this back in at line 15 - 20 to exit if XMLSecTool is missing.